supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesInitializationActor.scala

```diff
@@ -0,0 +1,157 @@
+package cromwell.backend.impl.jes
+
+import akka.actor.Props
+import cromwell.backend.BackendLifecycleActor.WorkflowAbortResponse
+import cromwell.backend.impl.jes.JesInitializationActor._
+import cromwell.backend.impl.jes.io.{JesAttachedDisk, JesWorkingDisk}
+import cromwell.backend.validation.RuntimeAttributesKeys._
+import cromwell.backend.validation.Validation._
+import cromwell.backend.validation.exception.ValidationAggregatedException
+import cromwell.backend.validation.{ContinueOnReturnCode, ContinueOnReturnCodeSet}
+import cromwell.backend.{BackendConfigurationDescriptor, BackendWorkflowDescriptor, BackendWorkflowInitializationActor}
+import cromwell.core._
+import lenthall.exception.AggregatedException
+import wdl4s.parser.MemoryUnit
+import wdl4s.types.{WdlIntegerType, WdlStringType}
+import wdl4s.values.{WdlArray, WdlInteger, WdlString, WdlValue}
+import wdl4s.{Call, RuntimeAttributes, WdlExpression}
+
+import scala.concurrent.Future
+import scalaz.Scalaz._
+
+object JesInitializationActor {
+  //TODO: check if these need to be configurable.
```

Comment:
Not necessarily right here but prior to us wrapping up it'd be nice if we had a cleaner pattern to handle this than spamming a bunch of junk at the top